images/base-images/nodejs6/Dockerfile

```diff
@@ -1,12 +1,22 @@
-FROM vmware/photon:2.0
+FROM vmware/photon2:20180302
 
-RUN tdnf install -y nodejs-7.7.4-4.ph2
+RUN tdnf install -y nodejs-8.3.0-1.ph2
```

Comment:
Should we call this image nodejs8-base instead? or just drop the version number. this "nodejs6" thing is kind of misleading now.